tests/js/client/load-balancing/load-balancing-pregel-auth-cluster.js

```diff
@@ -179,33 +189,32 @@ function PregelAuthSuite () {
           resultField: "result"
         }
       };
-      let result = sendRequest(users[0], 'POST', url, task, true);
-
-      assertFalse(result === undefined || result === {});
-      assertEqual(result.status, 200);
-      assertTrue(result.body > 1);
-
-      const taskId = result.body;
-      url = `${baseUrl}/${taskId}`;
-      result = sendRequest(users[1], 'GET', url, {}, false);
-
-      assertFalse(result === undefined || result === {});
-      assertTrue(result.body.error);
-      assertEqual(result.status, 404);
-
-      url = `${baseUrl}/${taskId}`;
-      result = sendRequest(users[1], 'DELETE', url, {}, {}, false);
-
-      assertFalse(result === undefined || result === {});
-      assertTrue(result.body.error);
-      assertEqual(result.status, 404);
-
-      url = `${baseUrl}/${taskId}`;
-      result = sendRequest(users[0], 'DELETE', url, {}, {}, false);
-
-      assertFalse(result === undefined || result === {});
-      assertFalse(result.body.error);
-      assertEqual(result.status, 200);
+      const postResponse = sendRequest(runCreator, 'POST', baseUrl, task, true);
+      assertFalse(postResponse === undefined || postResponse === {});
+      assertEqual(postResponse.status, 200);
+      const pid = postResponse.body;
+      assertTrue(pid > 1);
+
+      // need to wait for short time until pregel run was created
+      require("internal").sleep(1);
```

Comment:
Should we do the same retry loop here?
On slow/full Jenkins we may still see false negatives here just because we could not get CPU time.

Reply:
ah true
Mhm
Can we do the following:
We do the waiting loop on the correct user until it shows up (if this fails the test above needs to be red)
And then continue with the wrong user and expect it to be not found?

Reply:
so true, thanks!